Add test to install default starter and run "gatsby develop"

```
node --version
v4.4.5
```

Hi there, attempting to use gatsby for the first time, followed the README instructions to a T, and after inputting `gatsby develop`, received the following:

```
edouard:edouard edouard$ gatsby develop
Failed to require /Users/edouard/Projects/ed/projects/edouard/html.js
./html.js
Module not found: Error: Cannot resolve module 'babel' in /Users/edouard/Projects/ed/projects/edouard
resolve module babel in /Users/edouard/Projects/ed/projects/edouard
  looking for modules in /Users/edouard/Projects/ed/projects/edouard/loaders
    module variation babel-webpack-loader
      resolve 'file' babel-webpack-loader in /Users/edouard/Projects/ed/projects/edouard/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader doesn't exist (module as directory)
    module variation babel-web-loader
      resolve 'file' babel-web-loader in /Users/edouard/Projects/ed/projects/edouard/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader doesn't exist (module as directory)
    module variation babel-loader
      resolve 'file' babel-loader in /Users/edouard/Projects/ed/projects/edouard/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader doesn't exist (module as directory)
    module variation babel
      resolve 'file' babel in /Users/edouard/Projects/ed/projects/edouard/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/loaders/babel.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/loaders/babel doesn't exist (module as directory)
  looking for modules in /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders
    module variation babel-webpack-loader
      resolve 'file' babel-webpack-loader in /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader doesn't exist (module as directory)
    module variation babel-web-loader
      resolve 'file' babel-web-loader in /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader doesn't exist (module as directory)
    module variation babel-loader
      resolve 'file' babel-loader in /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader doesn't exist (module as directory)
    module variation babel
      resolve 'file' babel in /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel doesn't exist (module as directory)
  looking for modules in /Users/edouard/Projects/ed/projects/edouard/node_modules
    module variation babel-webpack-loader
      resolve 'file' babel-webpack-loader in /Users/edouard/Projects/ed/projects/edouard/node_modules
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader doesn't exist (module as directory)
    module variation babel-web-loader
      resolve 'file' babel-web-loader in /Users/edouard/Projects/ed/projects/edouard/node_modules
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader doesn't exist (module as directory)
    module variation babel-loader
      resolve 'file' babel-loader in /Users/edouard/Projects/ed/projects/edouard/node_modules
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader doesn't exist (module as directory)
    module variation babel
      resolve 'file' babel in /Users/edouard/Projects/ed/projects/edouard/node_modules
        resolve file
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel.webpack-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel.web-loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel.loader.js doesn't exist
          /Users/edouard/Projects/ed/projects/edouard/node_modules/babel.js doesn't exist
      /Users/edouard/Projects/ed/projects/edouard/node_modules/babel doesn't exist (module as directory)
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-webpack-loader]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-web-loader]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel-loader]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel.js]
[/Users/edouard/Projects/ed/projects/edouard/loaders/babel]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-webpack-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-web-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/gatsby/dist/loaders/babel]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-webpack-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-web-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel-loader]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel.webpack-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel.web-loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel.loader.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel.js]
[/Users/edouard/Projects/ed/projects/edouard/node_modules/babel]
 @ ./html.js 13:18-45
```

--- a/test/integration/install-starter.js
+++ b/test/integration/install-starter.js
@@ -1,0 +1,14 @@
+import test from 'ava'
+import fs from 'fs-extra'
+import Promise from 'bluebird'
+const remove = Promise.promisify(fs.remove)
+const tmpdir = require('os').tmpdir()
+
+import { gatsby } from '../support'
+
+test('calling gatsby new succesfully creates new site from default starter', async t => {
+  await remove(`${tmpdir}/gatsby-default-starter`)
+  const noArgs = await gatsby(['new', `${tmpdir}/tmp/gatsby-default-starter`])
+  console.log(noArgs)
+  t.is(noArgs.code, 0)
+})

